RadClass/DiffSpectra.py

```diff
@@ -0,0 +1,44 @@
+import numpy as np
+
+
+class DiffSpectra:
+    '''
+    Estimates a minimum background spectrum for each
+    timestamp/spectrum and saves a difference spectrum
+    between the two.
+
+    Attributes:
+    stride: The number of timestamps prior to the current
+        spectrum for finding the minimum background
+    '''
+
+    def __init__(self, stride=10):
+        self.stride = stride
+
+    def run(self, data):
+        # initializing spectra, with bckg to be subtracted below
+        self.diff_spectra = data[self.stride:].copy()
+        # compute all integrated count-rates
+        sums = np.sum(data[:, 1:], axis=1)
+        for i in range(self.diff_spectra.shape[0]):
+            # select the smallest spectrum from a window self.stride long
```

Comment:
This may warrant a longer comment since there is a lot going on here.
- there is a nuanced difference between `i` in `data` and in `diff_spectra` and in `sums`
- there is a window related to that nuanced difference
- the result of `argmin` is shifted and needs to incremented by `i`